### PR TITLE
Stats updates for 4.0

### PR DESCRIPTION
**Pull request by bsipocz**
Last PR for the release, to add contributor stats to the release notes, cleanup mailmap, etc.

### CODE REVIEW

**Comment by pllim** [comment]
`docs/whatsnew/4.0.rst:625:.. hlist content is not a list`

File: docs/whatsnew/4.0.rst [hunk 1]
```diff
@@ -617,3 +617,86 @@ Full change log
...
 API, please see the :ref:`changelog`.
+
+
+Contributors to the v4.0 release
+================================
+
+.. hlist::
+  :columns: 4
+
+
```
Comment (pllim) [inline docs/whatsnew/4.0.rst]: Would removing extra blank line fix the error?
```suggestion
```

**Comment by bsipocz** [comment]
I keep this open until actual release  time, so the numbers can be updated if needed.

**Comment by eteq** [comment]
Note that after some discussion with @bsipocz and @astrofrog, we agreed to add e0bc3a0 to this PR since it would otherwise require additional CI and thereby delay the release.  But it's

**Comment by bsipocz** [comment]
I've cancelled travis and will merge this once the docs build  is passing.

**Pull request merged**